scripts/plotting/postfitPlots.py

```diff
@@ -190,7 +190,7 @@ def make_plot(h_data, h_inclusive, h_stack, axes, colors=None, labels=None, suff
 
     scale = max(1, np.divide(*ax1.get_figure().get_size_inches())*0.3)
 
-    fontsize = ax1.xaxis.label.get_size()
+    fontsize = ax1.xaxis.label.get_size()*0.7
```

Comment:
(this may come from me) It would be good to avoid ad-hoc scaling of text size, this leads to inconsistent use.